astropy/modeling/tests/example_models.py

```diff
@@ -184,6 +184,15 @@
         'x_values': [1, 10, 100],
         'y_values': [3, 111, 10101],
         'x_lim': [-3, 3]
+     },
+
+    Sersic1D: {
+    'parameters': [1, 20, 4],
+    'x_values': [0.1, 1, 10, 100],
+    'y_values': [2.78629391e+02,   5.69791430e+01,   3.38788244e+00,
+                 2.23941982e-02],
+    'requires_scipy': True,
+    'x_lim': [-100,100]
```

Comment:
More substantive--the hanging of the tests in Windows is actually due to this line right here.  The test model is being evaluated on the range `[-100, 100]`.  But if I understand correctly this model is undefined for x <= 0, so the lower half of the test data is NaNs.  For some reason the fitter is hanging on Windows when given NaNs, which is probably an entirely separate issue unrelated to this model, I would guess.

For now just change this to `'x_lim': [0, 100]` and everything should work.  I'll open a separate issue for the fitter hanging on NaNs, which I have yet to debug.